cli/tests/unit/remove_test.ts

```diff
@@ -460,6 +483,29 @@ unitTest({ perms: { write: false } }, async function removeAllPerm(): Promise<
   assertEquals(err.name, "PermissionDenied");
 });
 
+unitTest(
+  {
+    ignore: Deno.build.os === "windows",
+    perms: { write: true, read: true },
+  },
+  async function removeUnixSocketSuccess(): Promise<void> {
+    // MAKE TEMPORARY UNIX SOCKET
+    const path = Deno.makeTempDirSync() + "/test.sock";
+    const listener = Deno.listen({ transport: "unix", path });
+    listener.close();
+    Deno.statSync(path); // check if unix socket exists
+
+    await Deno.remove(path);
+    let err;
+    try {
+      Deno.statSync(path);
+    } catch (e) {
+      err = e;
+    }
+    assert(err instanceof Deno.errors.NotFound);
+  }
+);
```

Comment:
Why are there two copies of the tests?

Reply:
I'm sorry, it's a bit confusing.
One is a test of `Deno.removeSync` and the other is a test of `Deno.remove`.

Reply:
Ah sorry, I didn't catch that. This is fine too. Thanks.